app/helpers/interactive_run_helper.rb

```diff
@@ -56,15 +56,17 @@ def interactive_iframe_tag(interactive,run=nil,iframe_src=true)
       url = interactive_run.learner_url
     end
     url = interactive.url if url.blank?
-    
+
     data = {
       :aspect_ratio => interactive.aspect_ratio,
       :id => interactive.id,
-      :iframe_mouseover => "false"
+      :iframe_mouseover => "false",
```

Comment:
During development I had some other data in there so I added the comma.  I'll remove it.